openshift_images/postgresql.adoc

```diff
@@ -10,89 +10,154 @@
 ====
 
 == Installation
-To build PostgreSQL image, choose between the CentOS7 or RHEL7 base image:
-
-=== RHEL-7 base image
-
-----
-$ git clone https://github.com/openshift/postgresql.git
-$ cd postgresql
-$ make build TARGET=rhel7 VERSION=5.5
-----
-
-=== CentOS-7 base image
-
-----
-$ git clone https://github.com/openshift/postgresql.git
-$ cd postgresql
-$ make build VERSION=5.5
-----
+To build a PostgreSQL image, use one of the following procedures, depending on
+your base image:
+
+*RHEL-7 Base Image*
+
+To build a RHEL-7 based PostgreSQL image:
+
+. Clone the following repository:
++
+****
+`$ git clone https://github.com/openshift/postgresql.git`
+****
+. Navigate into the directory you just created:
++
+****
+`$ cd postgresql`
+****
+. Build the image:
++
+****
+`$ make build TARGET=rhel7 VERSION=5.5`
+****
+
+*CentOS-7 Base Image*
+
+To build a CentOS-7 based PostgreSQL image:
+
+. Clone the following repository:
++
+****
+`$ git clone https://github.com/openshift/postgresql.git`
+****
+. Navigate into the directory you just created:
++
+****
+`$ cd postgresql`
+****
+. Build the image:
++
+****
+`$ make build VERSION=5.5`
+****
 
 [NOTE]
 ====
-By omitting the `VERSION` parameter, the build action will be performed on all the supported versions of PostgreSQL. Since we are now supporting only version `9.2`, you can omit this parameter.
+If you omit the `VERSION` parameter, then the build action is performed on all
+the supported versions of PostgreSQL. Currently, only version 9.2 is supported,
+so you can omit this parameter.
 ====
 
 == Usage
-To create a container running PostgreSQL, you need to run following commands.
+To create a container running PostgreSQL, you must run the following commands.
+
+*RHEL-7 image*
 
-=== RHEL-7 image
+****
+`$ docker run -d --name _<database>_ -e POSTGRESQL_USER=_<user>_ -e POSTGRESQL_PASSWORD=_<password>_ -e POSTGRESQL_DATABASE=_<database>_ -p 5432:5432 openshift/postgresql-92-rhel7`
+****
 
-----
-$ docker run -d --name postgresql_database -e POSTGRESQL_USER=user -e POSTGRESQL_PASSWORD=pass -e POSTGRESQL_DATABASE=db -p 5432:5432 openshift/postgresql-92-rhel7
-----
+*CentOS-7 image*
 
-=== CentOS-7 image
+****
+`$ docker run -d --name _<database>_ -e POSTGRESQL_USER=_<user>_ -e POSTGRESQL_PASSWORD=_<password>_ -e POSTGRESQL_DATABASE=_<database>_ -p 5432:5432 openshift/postgresql-92-centos7`
+****
 
-----
-$ docker run -d --name postgresql_database -e POSTGRESQL_USER=user -e POSTGRESQL_PASSWORD=pass -e POSTGRESQL_DATABASE=db -p 5432:5432 openshift/postgresql-92-centos7
-----
+== Testing
+To access and test if the application is running:
 
-This will create a container named `postgresql_database`, with credentials `user:pass` and database `db`. To access and test if the application is running, run:
-----
-$ psql postgresql://user:pass@127.0.0.1:5432/db <<< "SELECT 1;"
-----
+****
+`$ psql postgresql://_<user>_:<password>@127.0.0.1:5432/_<database>_ <<< "SELECT 1;"`
```

Comment:
Maybe we could use the example without passing the password on a command line and instead enter it in the prompt? Since using the command line is generally considered insecure, we probably shouldn't give a bad example.